pkg/util/config_util.go

```diff
@@ -49,6 +50,24 @@ func GetFromFile(c interface{}, filename string) error {
 	return nil
 }
 
+//WriteToJSONFile writes a struct to json file
+func WriteToJSONFile(c interface{}, filename string) error {
+	data, err := json.Marshal(c)
+	if err != nil {
+		return fmt.Errorf("unable to marshal odo config data: %w", err)
+	}
+
+	if err = CreateIfNotExists(filename); err != nil {
+		return err
+	}
+	err = ioutil.WriteFile(filename, data, 0600)
+	if err != nil {
+		return fmt.Errorf("unable to write config to file %v: %w", c, err)
+	}
+
+	return nil
+}
+
 // WriteToFile marshals a struct to a file
 func WriteToFile(c interface{}, filename string) error {
```

Comment:
Could you rename this one to `WriteToYAMLFile`?

Reply:
ok